cpp/include/raft/stats/contingency_matrix.cuh

```diff
@@ -19,6 +19,10 @@
 
 #pragma once
 
+#include <raft/core/device_mdarray.hpp>
```

Comment:
I don't think we need the `device_mdarray` includes in any  of these files unless they are explicitly calling `make_device_*` (without `_view`).

Reply:
`make_device_vector` is used here in this file to allocate a workspace.